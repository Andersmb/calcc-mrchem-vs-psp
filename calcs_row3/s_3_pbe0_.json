{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            0.10360312406116091
          ],
          "radius": 1.09
        },
        {
          "center": [
            0.0,
            1.81716051,
            -1.6479838959388389
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.0,
            -1.81716051,
            -1.6479838959388389
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "s",
          "xyz": [
            0.0,
            0.0,
            0.10360312406116091
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            1.81716051,
            -1.6479838959388389
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            -1.81716051,
            -1.6479838959388389
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "s_3_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        -8.378291362178759e-20,
        -2.3007166518016302e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.39762482624127077,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -2.8425782977603914e-07,
            6.720362465423171e-08,
            -0.39762482624116346
          ],
          "vector_el": [
            -2.843579535172225e-07,
            6.711440313516995e-08,
            1.2406929805810132
          ],
          "vector_nuc": [
            1.0012374118334261e-10,
            8.922151906176623e-11,
            -1.6383178068221766
          ]
        }
      },
      "geometry": [
        {
          "symbol": "S",
          "xyz": [
            0.0,
            0.0,
            0.10360312406116091
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            1.81716051,
            -1.6479838959388389
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            -1.81716051,
            -1.6479838959388389
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -89.02801246284857,
          -8.009972457889079,
          -5.946365525350004,
          -5.943594266872014,
          -5.938360746866983,
          -0.7652965097759422,
          -0.45973249231501023,
          -0.38330904845669783,
          -0.27798541818911815,
          -89.02801246284783,
          -8.009972457904398,
          -5.946365525353189,
          -5.943594266885339,
          -5.938360746861789,
          -0.7652965097903787,
          -0.4597324923309283,
          -0.38330904848881125,
          -0.2779854181822911
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -233.50525785720833
      },
      "scf_energy": {
        "E_ee": 190.95697966962345,
        "E_eext": 0.0,
        "E_el": -412.23016794566774,
        "E_en": -975.6406663402837,
        "E_kin": 398.7170366684962,
        "E_next": 0.0,
        "E_nn": 12.953887402931901,
        "E_nuc": 12.953887402931901,
        "E_tot": -399.27628054273583,
        "E_x": -6.424806399047544,
        "E_xc": -19.838711544456185,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 191.74997738233625,
        "E_eext": 0.0,
        "E_el": -410.18844965488057,
        "E_en": -972.3293393357371,
        "E_kin": 396.6577019776463,
        "E_next": 0.0,
        "E_nn": 12.953887402931901,
        "E_nuc": 12.953887402931901,
        "E_tot": -397.23456225194866,
        "E_x": -6.427789585264149,
        "E_xc": -19.839000093861923,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 190.3752625714192,
              "E_eext": 0.0,
              "E_el": -412.15601406148863,
              "E_en": -971.4420058070187,
              "E_kin": 395.08767681535943,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2021266585567,
              "E_x": -6.405168305033592,
              "E_xc": -19.771779336214955,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2021266585567,
            "energy_update": 1.9675644066080622,
            "mo_residual": 0.20196038064293334,
            "wall_time": 87.309046454
          },
          {
            "energy_terms": {
              "E_ee": 190.83296649925768,
              "E_eext": 0.0,
              "E_el": -412.2168504573959,
              "E_en": -973.9331860256135,
              "E_kin": 397.11337150022865,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.26296305446397,
              "E_x": -6.417024260836416,
              "E_xc": -19.812978170432363,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.26296305446397,
            "energy_update": 0.06083639590724488,
            "mo_residual": 0.06502322137578499,
            "wall_time": 62.477540462
          },
          {
            "energy_terms": {
              "E_ee": 190.83344133819213,
              "E_eext": 0.0,
              "E_el": -412.22612063089076,
              "E_en": -974.7832049132132,
              "E_kin": 397.9654671236753,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27223322795885,
              "E_x": -6.419493886847705,
              "E_xc": -19.822330292697316,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27223322795885,
            "energy_update": 0.009270173494883238,
            "mo_residual": 0.026980699926772638,
            "wall_time": 56.384253279
          },
          {
            "energy_terms": {
              "E_ee": 190.94466740947865,
              "E_eext": 0.0,
              "E_el": -412.22898419427884,
              "E_en": -975.4630796407874,
              "E_kin": 398.5473911125335,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2750967913469,
              "E_x": -6.423254066094931,
              "E_xc": -19.834709009408616,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2750967913469,
            "energy_update": 0.002863563388075363,
            "mo_residual": 0.021502505221592295,
            "wall_time": 56.413049988
          },
          {
            "energy_terms": {
              "E_ee": 190.97821393504938,
              "E_eext": 0.0,
              "E_el": -412.229911982297,
              "E_en": -975.8550689352422,
              "E_kin": 398.9132852913795,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27602457936507,
              "E_x": -6.425244531588836,
              "E_xc": -19.841097741894913,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27602457936507,
            "energy_update": 0.0009277880181457476,
            "mo_residual": 0.017981820809232697,
            "wall_time": 54.685033573
          },
          {
            "energy_terms": {
              "E_ee": 190.96602327980702,
              "E_eext": 0.0,
              "E_el": -412.23010058878856,
              "E_en": -975.7266241953138,
              "E_kin": 398.7951981021171,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27621318585665,
              "E_x": -6.4249693171417235,
              "E_xc": -19.83972845825715,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27621318585665,
            "energy_update": 0.00018860649157659282,
            "mo_residual": 0.007926377774660515,
            "wall_time": 56.575050305
          },
          {
            "energy_terms": {
              "E_ee": 190.96320080095228,
              "E_eext": 0.0,
              "E_el": -412.23015683719933,
              "E_en": -975.6467813772837,
              "E_kin": 398.71735882732554,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2762694342674,
              "E_x": -6.424893722229113,
              "E_xc": -19.83904136596433,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2762694342674,
            "energy_update": 5.624841077178644e-05,
            "mo_residual": 0.004015585746662687,
            "wall_time": 56.270979102
          },
          {
            "energy_terms": {
              "E_ee": 190.95791099381654,
              "E_eext": 0.0,
              "E_el": -412.2301647187355,
              "E_en": -975.632356118806,
              "E_kin": 398.7076669922836,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2762773158036,
              "E_x": -6.424768772054502,
              "E_xc": -19.838617813975073,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2762773158036,
            "energy_update": 7.88153619168952e-06,
            "mo_residual": 0.0013173186734633854,
            "wall_time": 57.833881506
          },
          {
            "energy_terms": {
              "E_ee": 190.9556626858539,
              "E_eext": 0.0,
              "E_el": -412.23016693115056,
              "E_en": -975.6285987731133,
              "E_kin": 398.7060085546845,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27627952821865,
              "E_x": -6.424741731454108,
              "E_xc": -19.83849766712165,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27627952821865,
            "energy_update": 2.2124150405034015e-06,
            "mo_residual": 0.0007299722657013904,
            "wall_time": 57.960691681
          },
          {
            "energy_terms": {
              "E_ee": 190.95574857772024,
              "E_eext": 0.0,
              "E_el": -412.2301675525092,
              "E_en": -975.636106553797,
              "E_kin": 398.7135698818944,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2762801495773,
              "E_x": -6.42477796250157,
              "E_xc": -19.83860149582517,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2762801495773,
            "energy_update": 6.213586516423675e-07,
            "mo_residual": 0.0004478362149313329,
            "wall_time": 58.492956557
          },
          {
            "energy_terms": {
              "E_ee": 190.95665451651743,
              "E_eext": 0.0,
              "E_el": -412.23016785834585,
              "E_en": -975.6440126535098,
              "E_kin": 398.7207439898102,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27628045541394,
              "E_x": -6.4248177232619925,
              "E_xc": -19.83873598790177,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27628045541394,
            "energy_update": 3.058366360164655e-07,
            "mo_residual": 0.00044344288545195986,
            "wall_time": 56.740339871
          },
          {
            "energy_terms": {
              "E_ee": 190.95701465421993,
              "E_eext": 0.0,
              "E_el": -412.23016792044524,
              "E_en": -975.6427934602642,
              "E_kin": 398.7191654867007,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.2762805175133,
              "E_x": -6.424816615155808,
              "E_xc": -19.838737985945908,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.2762805175133,
            "energy_update": 6.209938874235377e-08,
            "mo_residual": 0.0001568192279879176,
            "wall_time": 60.798830879
          },
          {
            "energy_terms": {
              "E_ee": 190.9570920200689,
              "E_eext": 0.0,
              "E_el": -412.2301679341231,
              "E_en": -975.6416209391055,
              "E_kin": 398.7179050917224,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27628053119116,
              "E_x": -6.42481334302436,
              "E_xc": -19.83873076378456,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27628053119116,
            "energy_update": 1.3677833976544207e-08,
            "mo_residual": 0.00013141062044558878,
            "wall_time": 60.778062355
          },
          {
            "energy_terms": {
              "E_ee": 190.95697966962345,
              "E_eext": 0.0,
              "E_el": -412.23016794566774,
              "E_en": -975.6406663402837,
              "E_kin": 398.7170366684962,
              "E_next": 0.0,
              "E_nn": 12.953887402931901,
              "E_nuc": 12.953887402931901,
              "E_tot": -399.27628054273583,
              "E_x": -6.424806399047544,
              "E_xc": -19.838711544456185,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.27628054273583,
            "energy_update": 1.1544670996954665e-08,
            "mo_residual": 9.312795032414382e-05,
            "wall_time": 66.762645815
          }
        ],
        "wall_time": 849.536914769
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
