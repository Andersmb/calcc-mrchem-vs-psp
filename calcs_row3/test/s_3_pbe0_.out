                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.103603
    1       H            :        0.000000        1.817161       -1.647984
    2       H            :        0.000000       -1.817161       -1.647984
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               9
 Beta electrons          :                                               9
 Total electrons         :                                              18
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999780245e-01
    1    1    a          :                              9.999998846873e-01
    2    1    a          :                              1.000000219894e+00
    3    1    a          :                              1.000000197038e+00
    4    1    a          :                              1.000000012651e+00
    5    1    a          :                              9.999992477850e-01
    6    1    a          :                              1.000002046446e+00
    7    1    a          :                              1.000002059484e+00
    8    1    a          :                              1.000000007962e+00
    9    1    b          :                              9.999999780245e-01
   10    1    b          :                              9.999998846873e-01
   11    1    b          :                              1.000000219894e+00
   12    1    b          :                              1.000000197038e+00
   13    1    b          :                              1.000000012651e+00
   14    1    b          :                              9.999992477850e-01
   15    1    b          :                              1.000002046446e+00
   16    1    b          :                              1.000002059488e+00
   17    1    b          :                              1.000000007956e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                396.657701977646
 E-N energy              :            (au)               -972.329339335737
 Coulomb energy          :            (au)                191.749977382336
 Exchange energy         :            (au)                 -6.427789585264
 X-C energy              :            (au)                -19.839000093862
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 12.953887402932
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -410.188449654881
 Nuclear energy          :            (au)                 12.953887402932
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.972345622519e+02
                         :      (kcal/mol)             -2.492684492272e+05
                         :        (kJ/mol)             -1.042939192243e+06
                         :            (eV)             -1.080930262861e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.300999491150
    1    1    a          :            (au)                 -7.863583463120
    2    1    a          :            (au)                 -5.804838563871
    3    1    a          :            (au)                 -5.800713881117
    4    1    a          :            (au)                 -5.792717020470
    5    1    a          :            (au)                 -0.756992939180
    6    1    a          :            (au)                 -0.451415977594
    7    1    a          :            (au)                 -0.329790900294
    8    1    a          :            (au)                 -0.236927474334
    9    1    b          :            (au)                -88.301001650651
   10    1    b          :            (au)                 -7.863583361827
   11    1    b          :            (au)                 -5.804838522572
   12    1    b          :            (au)                 -5.800714372931
   13    1    b          :            (au)                 -5.792717011295
   14    1    b          :            (au)                 -0.756992923012
   15    1    b          :            (au)                 -0.451415975806
   16    1    b          :            (au)                 -0.329787374065
   17    1    b          :            (au)                 -0.236927475146
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -230.675958378435
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.242641e+00        -397.234562251949         -3.972346e+02
    1          2.019604e-01        -399.202126658557         -1.967564e+00
    2          6.502322e-02        -399.262963054464         -6.083640e-02
    3          2.698070e-02        -399.272233227959         -9.270173e-03
    4          2.150251e-02        -399.275096791347         -2.863563e-03
    5          1.798182e-02        -399.276024579365         -9.277880e-04
    6          7.926378e-03        -399.276213185857         -1.886065e-04
    7          4.015586e-03        -399.276269434267         -5.624841e-05
    8          1.317319e-03        -399.276277315804         -7.881536e-06
    9          7.299723e-04        -399.276279528219         -2.212415e-06
   10          4.478362e-04        -399.276280149577         -6.213587e-07
   11          4.434429e-04        -399.276280455414         -3.058366e-07
   12          1.568192e-04        -399.276280517513         -6.209939e-08
   13          1.314106e-04        -399.276280531191         -1.367783e-08
   14          9.312795e-05        -399.276280542736         -1.154467e-08
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.103603
    1       H            :        0.000000        1.817161       -1.647984
    2       H            :        0.000000       -1.817161       -1.647984
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                398.717036668496
 E-N energy              :            (au)               -975.640666340284
 Coulomb energy          :            (au)                190.956979669623
 Exchange energy         :            (au)                 -6.424806399048
 X-C energy              :            (au)                -19.838711544456
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 12.953887402932
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -412.230167945668
 Nuclear energy          :            (au)                 12.953887402932
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.992762805427e+02
                         :      (kcal/mol)             -2.505496467877e+05
                         :        (kJ/mol)             -1.048299722840e+06
                         :            (eV)             -1.086486061118e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.028012462849
    1    1    a          :            (au)                 -8.009972457889
    2    1    a          :            (au)                 -5.946365525350
    3    1    a          :            (au)                 -5.943594266872
    4    1    a          :            (au)                 -5.938360746867
    5    1    a          :            (au)                 -0.765296509776
    6    1    a          :            (au)                 -0.459732492315
    7    1    a          :            (au)                 -0.383309048457
    8    1    a          :            (au)                 -0.277985418189
    9    1    b          :            (au)                -89.028012462848
   10    1    b          :            (au)                 -8.009972457904
   11    1    b          :            (au)                 -5.946365525353
   12    1    b          :            (au)                 -5.943594266885
   13    1    b          :            (au)                 -5.938360746862
   14    1    b          :            (au)                 -0.765296509790
   15    1    b          :            (au)                 -0.459732492331
   16    1    b          :            (au)                 -0.383309048489
   17    1    b          :            (au)                 -0.277985418182
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -233.505257857208
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000000        1.240693
 Magnitude               :            (au)                        1.240693
                         :         (Debye)                        3.153527
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -1.638318
 Magnitude               :            (au)                        1.638318
                         :         (Debye)                        4.164188
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.000000       -0.397625
 Magnitude               :            (au)                        0.397625
                         :         (Debye)                        1.010661
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 15m 10s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
